client/my-sites/sharing/connections/services/facebook.js

```diff
@@ -0,0 +1,40 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import { SharingService, connectFor } from 'my-sites/sharing/connections/service';
+
+export class Facebook extends SharingService {
+	static propTypes = {
+		...SharingService.propTypes,
+	};
+
+	static defaultProps = {
+		...SharingService.defaultProps,
+	};
+
+	didKeyringConnectionSucceed( availableExternalAccounts ) {
```

Comment:
This looks good, but I think we should purposely disable it until August 1st, as per p9j7e4-gV-p2

Reply:
Note: We discussed this and it can be merged before August 1st.

Reply:
👍